pkg/util/containers/pause.go

```diff
@@ -8,11 +8,13 @@ package containers
 const containerNameLabel = "io.kubernetes.container.name"
 const podNameLabel = "io.kubernetes.pod.name"
 const pauseContainerNameValue = "POD"
+const sandboxLabelKey = "io.cri-containerd.kind"
+const sandboxLabelValue = "sandbox"
 
 // IsPauseContainer returns whether a container is a pause container based on the container labels
 // This util can be used to exclude pause container in best-effort
 // Note: Pause containers can still be excluded based on the image name via the container filtering module
-func IsPauseContainer(labels map[string]string) bool {
+func IsPauseContainer(labels map[string]string, imageName string, pauseContainerFilter *Filter) bool {
```

Comment:
could you update the function description since now the filter based on the image name also happen in this function. 🙇

Reply:
yes, ofc!